definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -5605,5 +5605,89 @@
     "CaseEventID": "cancelPronouncement",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solConfirmPersonalService",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solConfirmPersonalService",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solConfirmPersonalService",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solConfirmPersonalService",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "05/08/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "issuePersonalAos",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "05/08/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "issuePersonalAos",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "05/08/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "issuePersonalAos",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "05/08/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "issuePersonalAos",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "CRU"
+  },
```

Comment:
Is this permission correct? Solicitor should only have R on this if I'm not mistaken. They should not be able to issue their own case